.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+    "CodeGPT.apiKey": "Anthropic"

```

Comment:
Thank you so much for your perseverance and dedication, however, there is still a small change should be removed :)

Reply:
oh i'm sorry this is not suppose to happen, let me fix this.

Reply:
![image](https://github.com/langgenius/dify/assets/142148415/6c9c27b6-f8bb-46b0-beae-599c41c6964f)
should i just delete this whole folder for just this JSON string.

Reply:
Just remove the settings.json will make sense.